src/distro/distro.py

```diff
@@ -1062,6 +1062,13 @@ def _parse_os_release_content(lines: TextIO) -> Dict[str, str]:
                 # Ignore any tokens that are not variable assignments
                 pass
 
+        if "version" in props:
+            # extract release codename from the the version
+            match = re.search(r"(\(\D+\))|,(\s+)?\D+", props["version"])
+            if match:
+                # release codename appears within parentheses.
+                release_codename = match.group().strip(", ()")
```

Comment:
The previous effect of plain `.strip()` seems to have been dropped while moving. Is that intended?

Reply:
To be more precise: `.strip()` drops whitespace other than spaces. If we only ever have space, we'd be good, but `\s` is more than spaces.

Reply:
Yes it was, we are not _supposed_ to encounter whitespaces other than spaces in os-release files. While de-duplicating `strip` calls, I preferred expliciting the list of characters we want to drop purposely. I can revert to the previous behavior if you prefer :+1:

Reply:
I'm mostly the messager here. If you want to go with all spaces, then `(\s+)?` probably should become `( *)` (a space and an asterisk).

Reply:
Oh man, I mean as much as much as like non-capturing parentheses, I'm not sure this is the best way forward :joy:

Reply:
See updated branch ! I opted for two distinct matching groups, so no extra parsing nor non-capturing groups are required :innocent: